doc/project/release_process.rst

```diff
@@ -45,6 +45,10 @@ The Zephyr release model was loosely based on the Linux kernel model:
 
     Release Cycle
 
+.. note::
+
+    The milestones for the current major version can be found on the `Official GitHub Wiki <https://github.com/zephyrproject-rtos/zephyr/wiki/Release-Management>`_. Information on previous releases can be found :ref:`here <zephyr_release_notes>`.
```

Comment:
hey thanks for adding this, could you wrap this to 80 columns like the rest of the file?

Reply:
Done